<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="3" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-05-16T15:37:20 GST" name="Suite" finished-at="2025-05-16T15:37:47 GST" duration-ms="27412">
    <groups>
      <group name="sanity">
        <method signature="TC01_User_Registration.Registration()[pri:0, instance:testCases.TC01_User_Registration@313ac989]" name="Registration" class="testCases.TC01_User_Registration"/>
        <method signature="TC03_AddtoCart.add_cart()[pri:0, instance:testCases.TC03_AddtoCart@2a4fb17b]" name="add_cart" class="testCases.TC03_AddtoCart"/>
        <method signature="TC02_SignIn.Invalidcred()[pri:1, instance:testCases.TC02_SignIn@34a3d150]" name="Invalidcred" class="testCases.TC02_SignIn"/>
        <method signature="TC02_SignIn.user_login()[pri:2, instance:testCases.TC02_SignIn@34a3d150]" name="user_login" class="testCases.TC02_SignIn"/>
      </group> <!-- sanity -->
    </groups>
    <test started-at="2025-05-16T15:37:20 GST" name="Test" finished-at="2025-05-16T15:37:47 GST" duration-ms="27412">
      <class name="testCases.TC01_User_Registration">
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.TC01_User_Registration@313ac989]" started-at="2025-05-16T15:37:20 GST" name="setup" finished-at="2025-05-16T15:37:25 GST" duration-ms="5360" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="Registration()[pri:0, instance:testCases.TC01_User_Registration@313ac989]" started-at="2025-05-16T15:37:25 GST" name="Registration" finished-at="2025-05-16T15:37:29 GST" duration-ms="3889" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [https://petstore.octoperf.com/actions/Catalog.action] but found [https://petstore.octoperf.com/actions/Account.action]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [https://petstore.octoperf.com/actions/Catalog.action] but found [https://petstore.octoperf.com/actions/Account.action]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at testCases.TC01_User_Registration.Registration(TC01_User_Registration.java:81)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Registration -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.TC01_User_Registration@313ac989]" started-at="2025-05-16T15:37:29 GST" name="tearDown" finished-at="2025-05-16T15:37:29 GST" duration-ms="111" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testCases.TC01_User_Registration -->
      <class name="testCases.TC03_AddtoCart">
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.TC03_AddtoCart@2a4fb17b]" started-at="2025-05-16T15:37:40 GST" name="setup" finished-at="2025-05-16T15:37:43 GST" duration-ms="3508" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="add_cart()[pri:0, instance:testCases.TC03_AddtoCart@2a4fb17b]" started-at="2025-05-16T15:37:43 GST" name="add_cart" finished-at="2025-05-16T15:37:47 GST" duration-ms="3307" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_cart -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.TC03_AddtoCart@2a4fb17b]" started-at="2025-05-16T15:37:47 GST" name="tearDown" finished-at="2025-05-16T15:37:47 GST" duration-ms="230" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testCases.TC03_AddtoCart -->
      <class name="testCases.TC02_SignIn">
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.TC02_SignIn@34a3d150]" started-at="2025-05-16T15:37:29 GST" name="setup" finished-at="2025-05-16T15:37:33 GST" duration-ms="3625" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="Invalidcred()[pri:1, instance:testCases.TC02_SignIn@34a3d150]" started-at="2025-05-16T15:37:33 GST" name="Invalidcred" finished-at="2025-05-16T15:37:35 GST" duration-ms="2492" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Invalidcred -->
        <test-method signature="user_login()[pri:2, instance:testCases.TC02_SignIn@34a3d150]" started-at="2025-05-16T15:37:35 GST" name="user_login" finished-at="2025-05-16T15:37:40 GST" duration-ms="4215" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- user_login -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.TC02_SignIn@34a3d150]" started-at="2025-05-16T15:37:40 GST" name="tearDown" finished-at="2025-05-16T15:37:40 GST" duration-ms="274" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testCases.TC02_SignIn -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
